src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/AsyncMethodNamesShouldEndInAsync.cs

```diff
@@ -35,6 +35,7 @@ public abstract class AsyncMethodNamesShouldEndInAsyncAnalyzer : DiagnosticAnaly
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.EnableConcurrentExecution();
```

Comment:
I am going to create a separate PR to delete all Async analyzers from the repo, which are anyways not implemented. Its fine to keep this change though, we can resolve conflicts as needed.